opds.py

```diff
@@ -42,16 +46,27 @@ def annotate_work_entry(cls, work, active_license_pool, edition, identifier, fee
     def default_lane_url(cls):
         return cdn_url_for("feed", _external=True)
 
+    def top_level_title(self):
+        return "All Books"
+
+    def lane_url(self, lane, facets=None, pagination=None):
+        if not facets:
+            facets = Facets.default()
+        if not pagination:
+            pagination = Pagination.default()
+        return self.feed_url(lane, facets, pagination)
+
     def feed_url(self, lane, facets, pagination):
         kwargs = dict(facets.items())
         kwargs.update(dict(pagination.items()))
-        if lane.license_source:
+        if lane and lane.license_source:
             view = "feed_from_license_source"
             kwargs['license_source_name'] = lane.license_source.name
         else:
             view = "feed"
-            kwargs['lane'] = lane.name
-            kwargs['languages'] = lane.languages
+            if lane:
+                kwargs['lane'] = lane.name
+                kwargs['languages'] = lane.languages
```

Comment:
These changes may not be necessary anymore with https://github.com/NYPL-Simplified/server_core/pull/208